packages/terra-worklist-data-grid/src/subcomponents/Row.jsx

```diff
@@ -110,10 +110,10 @@ function Row(props) {
 
   const columnIndexOffSet = hasRowSelection ? 1 : 0;
 
-  const handleCellSelect = (rowIdColId, coordinates) => {
-    if (hasRowSelection) {
+  const handleCellSelect = (rowIdColId, coordinates, isShiftKeyPressed, isCellSelectedFromKeyboard) => {
+    if (hasRowSelection || isShiftKeyPressed) {
```

Comment:
If we are not in row selection mode, why do we care if the shift key has been pressed?  Using shift outside row selection mode would multi-select cells not rows, correct?

Reply:
The requirements demand us to enable row selection mode if it is off for Shift + Up/Down arrow keys, Shift + Space and Shift + mouse click.